### PR TITLE
package_test.go: tiny fix to panic output

### PR DESCRIPTION
**Pull request by mattyw**

### CODE REVIEW

File: package_test.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func setLocationsForErrorTags(filename string) {
 		if j := strings.Index(line, "//err "); j >= 0 {
 			tag := line[j+len("//err "):]
 			if _, found := tagToLocation[tag]; found {
-				panic(fmt.Errorf("tag %q already processed previously"))
+				panic(fmt.Errorf("tag %q already processed previously", tag))
```
Comment (davecheney) [inline package_test.go]: Please make this fmt.Printf

**Comment by mattyw** [comment]
@davecheney I assume you meant Sprintf

**Comment by davecheney** [comment]
yup

**Comment by fwereade** [comment]
LGTM

**Comment by fwereade** [comment]
$$merge$$

**Pull request merged**